content/en/docs/Core concepts/concurrency.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Tuning concurrency"
+date: 2020-11-02
+weight: 3
+---
+
+Baker allows to tune concurrency at various levels of a pipeline:
+
+* input: Baker configuration doesn't expose knobs to tune input concurrency as it highly depends
+on the input source and how the input is implemented
+* filters: Baker runs N concurrent filter chains
+* output: Baker runs M concurrent outputs
+
+By default then, Baker processes records concurrently, without any guaranteed order.  
+However if you need to maintain the order of the records through the whole pipeline, it is still
+possible disabling concurrency ([see below for details](#guarantee-records-order)).
```

Comment:
maybe: 
possible to disable concurrency
or
possible by disabling concurrency
?